reporter/src/main/kotlin/reporters/ExcelReporter.kt

```diff
@@ -181,7 +191,7 @@ class ExcelReporter : TableReporter() {
         (0..1).forEach { sheet.autoSizeColumn(it) }
     }
 
-    private fun createSummarySheet(workbook: XSSFWorkbook, name: String, file: String, table: SummaryTable,
+    private fun createSummarySheet(workbook: XSSFWorkbook, name: String, file: String, table: TabularModelMapper.SummaryTable,
```

Comment:
Put the parameters on separate lines:
https://kotlinlang.org/docs/reference/coding-conventions.html#function-formatting